api/configuration/health.ts

```diff
@@ -0,0 +1,20 @@
+import {EnvironmentConfigServices} from '../interfaces/environment.config'
```

Comment:
Being picky, these should have spacing within the curly braces. Same for the line below.

Reply:
done